docs/machine-learning/tutorials/index.md

```diff
@@ -1,11 +1,13 @@
 ---
 title: ML.NET tutorials
 description: Explore these ML.NET tutorials to learn how to build custom AI solutions and integrate them into your .NET applications.
-ms.date: 05/07/2018
+ms.date: 06/06/2018
 ---
 # ML.NET tutorials
 
 The following tutorials enable you to understand how to use [ML.NET](../index.md) to build custom machine learning solutions and integrate them into your .NET applications:
 
 * [Sentiment Analysis](sentiment-analysis.md): demonstrates how to apply a binary classification task using ML.NET.
 * [Taxi Fare Predictor](taxi-fare.md): demonstrates how to apply a regression task using ML.NET.
+
+For more examples that use ML.NET, check the [dotnet/machinelearning-samples GitHub repository](https://github.com/dotnet/machinelearning-samples).
```

Comment:
one little nit: I usually like to keep just the repo name as the link as in `[dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples)  GitHub repository`